index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio setup mobile</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="container">
+    <header class="toolbar">
+      <a href="#" class="logo">My logo</a>
+      <nav class="navbar">
+        <div class="hambergur-button">
+          <img src="images/Icon - Menu.png" alt="hambergur-icon">
+        </div>
+      </nav>
+    </header>
+    <main class="main-content">
+      <div class="text-content">
+        <h1 class="heading-1">
+          I'm Lugard
+          <br>
+          Glad to see
+          <br>
+          you!
+        </h1>
+        <p class="paragraph">
+          I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h2 class="paragraph-2">LET'S CONNECT!</h2>
```

Comment:
- [ OPTIONAL ] I think here this need to be horizontal so you can add some font size or padding to the heading tag so please fix this 💯😊🚀